tools/dependencies/known-dependencies.txt

```diff
@@ -485,6 +485,7 @@ azure-storage-blob-12.21.0.jar
 azure-storage-internal-avro-12.6.0.jar
 vertica-jdbc-12.0.4-0.jar
 esdk-obs-java-bundle-3.23.3.jar
+cos_api-5.6.231.jar
```

Comment:
Missing license file.

Reply:
license for `cos_api-5.6.231.jar` is defined in `dolphinscheduler-dist/release-docs/LICENSE`:

```text
cos_api 5.6.231 https://mvnrepository.com/artifact/com.qcloud/cos_api/5.6.231, MIT
```